oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/RunningMode.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.core;
+
+import com.google.common.base.Strings;
+
+/**
+ * The running mode of the OAP server.
+ *
+ * @author wusheng
+ */
+public class RunningMode {
+    private static String MODE = "";
```

Comment:
the `MODE` should have a init value. Perhaps it's `no-init`.

Reply:
Default isn't `no-init` mode. It is just blank.